.pre-commit-config.yaml

```diff
@@ -68,6 +68,15 @@ repos:
           - flake8-simplify
           - flake8-type-checking
 
+-   repo: https://github.com/PyCQA/flake8
+    rev: 6.0.0
+    hooks:
+    -   id: flake8
+        args: ["--exclude", ".*,tests/eval_files/*"]
+        language_version: python3
+        additional_dependencies:
+          - flake8-pyi
```

Comment:
This looks identical to the above, can we just add to the `additional_dependencies` list?

Reply:
Oh oops, I didn't notice that got added to this PR. I did a cleanup of stashes and branches yesterday, and must've mixed up the playing-around-with-flake8-pyi stash/branch with the capitalize-errors stash/branch. I'll remove it from this PR and recheck what the end result of playing around with flake8-pyi was, there was some reason I didn't go ahead and push for its inclusion right away.

Reply:
Oh, and re it being in a separate environment:
> Note: Be careful when using this plugin in the same environment as other flake8 plugins, as they might generate errors that are inappropriate for .pyi files (e.g., about missing docstrings). We recommend running flake8-pyi in a dedicated environment in your CI.

https://github.com/PyCQA/flake8-pyi
(which clearly should be labeled with a comment in the config)